inbm/dispatcher-agent/dispatcher/source/constants.py

```diff
@@ -26,22 +26,22 @@ class SourceParameters:
 
 @dataclass(kw_only=True, frozen=True)
 class ApplicationAddSourceParameters:
-    file_name: str
-    sources: list[str] = field(default_factory=lambda: [])
+    source_list_file_name: str
```

Comment:
**Suggestion:** Replace list[str] with List[str] for type consistency across the codebase. [maintainability]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
@dataclass(kw_only=True, frozen=True)
class ApplicationSourceList:
    name: str
    sources: List[str]
```

Reply:
We're moving to use the new 'list' instead of having to add an import